ui/raidboss/data/04-sb/dungeon/bardams_mettle.js

```diff
@@ -0,0 +1,315 @@
+      },
+    },
+    {
+      id: 'Bardam\'s Mettle Meteor Impact',
+      netRegex: NetRegexes.startsUsing({ id: '2582', source: 'Looming Shadow' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '2582', source: 'Lauernd[a] Schatten' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '2582', source: 'ombre grandissante' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '2582', source: '落下地点' }),
+      netRegexCn: NetRegexes.startsUsing({ id: '2582', source: '坠落地点' }),
+      delaySeconds: function(data, matches) {
+        return parseFloat(matches.castTime) - 7;
+      },
+      alertText: {
+        en: 'Hide behind boulder',
+        de: 'Hinter dem Brocken verstecken',
+        fr: 'Cachez-vous derrière le rocher',
+        cn: '站在陨石后',
+      },
+    },
+    {
+      id: 'Bardam\'s Mettle Wind Unbound',
+      netRegex: NetRegexes.startsUsing({ id: '1F0A', source: 'Yol', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '1F0A', source: 'Yol', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '1F0A', source: 'Yol', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '1F0A', source: 'ヨル', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '1F0A', source: '胡鹰', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Bardam\'s Mettle Flutterfall',
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: function(data, matches) {
+        return data.me == matches.target && data.deadBardam;
+      },
+      response: Responses.spread(),
+    },
+    {
+      id: 'Bardam\'s Mettle Eye Of The Fierce',
+      netRegex: NetRegexes.startsUsing({ id: '1F0D', source: 'Yol', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '1F0D', source: 'Yol', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '1F0D', source: 'Yol', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '1F0D', source: 'ヨル', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '1F0D', source: '胡鹰', capture: false }),
+      response: Responses.lookAway(),
+    },
+    {
+      id: 'Bardam\'s Mettle Wingbeat You',
+      netRegex: NetRegexes.headMarker({ id: '0010' }),
+      condition: Conditions.targetIsYou(),
+      alertText: {
+        en: 'Knockback Laser on YOU',
+        de: 'Rückstoß-Laser auf DIR',
+      },
+    },
+    {
+      id: 'Bardam\'s Mettle Wingbeat Others',
+      netRegex: NetRegexes.headMarker({ id: '0010' }),
+      condition: Conditions.targetIsNotYou(),
+      infoText: {
+        en: 'Avoid Laser',
+        de: 'Laser ausweichen',
+      },
+    },
+  ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Yol': 'Yol',
+        'Warrior Of Bardam': 'Bardams Krieger',
+        'Throwing Spear': 'Wurfspeer',
+        'Star Shard': 'Sternensplitter',
+        'Looming Shadow': 'Lauernd[a] Schatten',
```

Comment:
please replace the `[a]` with `(?:e|er|es|en)` so the line should look like `'Looming Shadow': 'Lauernd(?:e|er|es|en) Schatten',`
Thanks in advance